rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/core/RestControllerHelper.java

```diff
@@ -603,6 +603,18 @@ public User convertToEmbeddedUser(User user) {
         return embeddedUser;
     }
 
+    public User convertToEmbeddedGetUsers(User user) {
```

Comment:
Why create new method when you can add the new fields in the above method(convertToEmbeddedUser).

Reply:
When I add new fields in method(convertToEmbeddedUser) it causes redundancy for about 11APIs using that function. It's not really necessary.